src/main/java/frc/robot/commands/CameraDrive.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+
+public class CameraDrive extends CommandBase {
```

Comment:
This command is unused, and looks like a duplicate of `CameraDriveCommand`. If it's not going to be used then delete it.